proxy/http2/Http2Stream.cc

```diff
@@ -767,10 +767,10 @@ Http2Stream::destroy()
 void
 Http2Stream::response_initialize_data_handling(bool &is_done)
 {
-  is_done           = false;
-  const char *name  = "transfer-encoding";
-  const char *value = "chunked";
-  int chunked_index = response_header.value_get_index(name, strlen(name), value, strlen(value));
+  is_done                                 = false;
+  static constexpr std::string_view value = "chunked";
+  int chunked_index =
+    response_header.value_get_index(TS_MIME_FIELD_TRANSFER_ENCODING, TS_MIME_LEN_TRANSFER_ENCODING, value.data(), value.size());
```

Comment:
Why not `TS_HTTP_VALUE_CHUNKED` and `TS_HTTP_LEN_CHUNKED`?

Reply:
I looked but missed those. Let me update.